setup.py

```diff
@@ -38,7 +38,7 @@ def read(*parts):
     'django>=1.5,<2.0',
     'babel>=1.3',
     'django-babel>=0.3.9',
-    'markey>=0.7',
+    'markey>=0.8',
```

Comment:
@EnTeQuAk this is expecting a version of `markey` that does not yet exist: https://github.com/EnTeQuAk/markey/releases